### PR TITLE
Convert import permissions to use Permissible

### PR DESCRIPTION
**Pull request by Throne3d**
We'll need to manually:

```ruby
User.where(id: [1, 2, 3, 8, 19, 24, 31], role_id: nil).update_attributes(role_id: Permissible::IMPORTER)
```

This should be neater than the current hardcoding method.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -88,7 +92,10 @@ def new
   end
 
   def create
-    import_thread and return if params[:button_import].present?
+    if params[:button_import].present?
```
Comment (Marri) [inline app/controllers/posts_controller.rb]: Why did this change?

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -375,6 +379,13 @@ def require_permission
     end
   end
 
+  def require_import_permission
```
Comment (Marri) [inline app/controllers/posts_controller.rb]: What if we just made this a before_action and returned unless view=import || button_import.present? Might be cleaner?

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (Throne3d) [reply app/controllers/posts_controller.rb]: `import_thread` stopped returning a truthy value all the time, so in some circumstances `import_thread and return` was failing to return.

I could just make this line: https://github.com/Marri/glowfic/pull/665/files#diff-ab4e9273f7df75aa5004a42f757d6f51R313 into `return true if performed?`?

File: app/controllers/posts_controller.rb (hunk 2 above)
Comment (Throne3d) [reply app/controllers/posts_controller.rb]: Done! Looks to solve the other comment too.

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (Throne3d) [reply app/controllers/posts_controller.rb]: (Solved by converting `require_import_permission` to a `before_action`.)

**Pull request merged**